RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelPhase2DigiToClusterCUDA.cc

```diff
@@ -87,8 +92,12 @@ void SiPixelPhase2DigiToClusterCUDA::fillDescriptions(edm::ConfigurationDescript
   edm::ParameterSetDescription desc;
 
   desc.add<bool>("IncludeErrors", true);
-  desc.add<int32_t>("clusterThreshold_layer1", kSiPixelClusterThresholdsDefaultPhase2.layer1);
-  desc.add<int32_t>("clusterThreshold_otherLayers", kSiPixelClusterThresholdsDefaultPhase2.otherLayers);
+  desc.add<int32_t>("clusterThreshold_layer1", 4000);
+  desc.add<int32_t>("clusterThreshold_otherLayers", 4000);
+  desc.add<double>("ElectronPerADCGain", 1500.f);
```

Comment:
nitpicking: since this is a `double` it should simply be `1500.` instead of `1500.f`